### ISSUE TITLE
Is there any shortcut to insert a new line below the current line?

### ISSUE THREAD

**Issue by GoBigorGoHome** [jupyter/notebook#3918]
As described in the title.

**Comment by hamcapem** [comment]
> esc b is a shortcut to add a new cell beneath the current one. Likewise, esc a inserts a new cell above the current one.

That's nice and all, but what about lines (not cells!)

**Comment by buzzf** [comment]
I have the same requirement。Inserting a new line now is not  that convenient like pycharm which has a  shortcut "shift + enter"

**Comment by efaves** [comment]
On a Mac you can use fn + return.

**Comment by Ahmed7fathi** [comment]
is there any solution yet ?

**Comment by dominic-springfield** [comment]
anyone?

**Comment by jazbassi1** [comment]
Yes you simply click on the area to the left of In [ ] until the box goes from Green to Blue, then Type 'a' for a new cell, on windows. Not sure if it is the same for mac

**Comment by hamcapem** [comment]
> Yes you simply click on the area to the left of In [ ] until the box goes from Green to Blue, then Type 'a' for a new cell, on windows. Not sure if it is the same for mac

Hahahaha.. how long have you been coding?

**Comment by tomato1mule** [comment]
just pressing 'end' key then 'enter' will do

**Comment by Ahmed7fathi** [comment]
> just pressing 'end' key then 'enter' will do

wow that's old gold smart way thank u

**Comment by vijayabhaarathy** [comment]
alt + enter will do

**Comment by wiktor-jurek** [comment]
Would really appreciate this functionality. In VSCode we have Shift-Enter and it saves a whole load of time when entering parameters.

**Comment by Tixotrop** [comment]
Windows:
ctrl + down (moving to the end of line)
enter

**Comment by SDZZGNDRC** [comment]
> alt + enter will do

It just 'Run Selected Cells and Insert a new Cell below', maybe 'insert a new line below in the same cell' will be better, just like Ctrl+Enter in VSCode.

**Comment by nianqinianyi** [comment]
ctrl + shift + enter

**Comment by marcndo** [comment]
I tried a majority of the mac os based proposals, unfortunately none worked for me.  I had to ensure that the two sentences i which to put in different lines are in saperate lines in a single cell then hit return to create an extra space between the sentences which worked. But this will have to create an empty line between the sentences.

**Comment by DaveSnell** [comment]
In Windows, you can use escape b to create a blank cell below the current
one, and escape a to create one above the current cell.

**Comment by karyeija** [comment]
## On Windows, specifically using vs code. 
1. Press cntrl+shift+P
2. Type "Insert Line Below" 
3. Click the ⚙️ icon
4. Double click in the selected area(a row)
5. Press your desired combination of keys
6. Press Enter (Return key)
**7.** You have successfully assigned key combination for your action. You can do the same for inserting line above

**Comment by kartheekkumarc** [comment]
> just pressing 'end' key then 'enter' will do

In Colab notebook, this worked for me 

- press 'Enter' key to insert a new line **in** the code-cell
